cmd/podman/play_kube.go

```diff
@@ -62,12 +62,13 @@ func init() {
 	flags.BoolVar(&playKubeCommand.TlsVerify, "tls-verify", true, "Require HTTPS and verify certificates when contacting registries")
 }
 
-func playKubeYAMLCmd(c *cliconfig.KubePlayValues) error {
+func playKubeYAMLCmd(c *cliconfig.KubePlayValues) (err error) {
```

Comment:
Nit:  I prefer to capatalize this err.  (Err error), then you can continue to use the lower case err as you always would.